### DISCUSSION TITLE
Unable to Install Pycuda on WSL2 with Intel Core

### DISCUSSION THREAD

**Discussion by AnushaManila** [inducer/pycuda#378] [Troubleshooting]
I’ve been trying to install Pycuda in the environment below:

* OS: Windows 10 Pro
* WSL version: 2
* Linux Distribution: Ubuntu 20.04
* GPU: Nvidia RTX A2000
* Driver Version: 512.15
* CUDA Version: 11.6
What I’ve tried and didn’t work:
1. pip install pycuda
Error: “Could not build wheels for pycuda”
2. Downloaded the wheel from [Archived: Python Extension Packages for Windows - Christoph Gohlke](https://www.lfd.uci.edu/~gohlke/pythonlibs/#pycuda)
Error: “pycuda-2022.1+cuda116-cp38-cp38-win_amd64.whl is not a supported wheel on this platform.”
The above link doesn’t have a file for Intel Core.
Is it possible to share a precompiled wheel for Intel Core 64-bit OS?

**Comment by inducer** [comment]
You could try downloading pycuda from conda forge.

**Comment by AnushaManila** [comment]
Issue solved by building Pycuda from source!
